modules/core/src/v2/coins/ofc.ts

```diff
@@ -102,7 +101,7 @@ export class Ofc extends BaseCoin {
     return Bluebird.resolve(true).asCallback(callback);
   }
 
-  signTransaction(params: SignTransactionOptions): SignedTransaction {
+  signTransaction(params: SignTransactionOptions): Bluebird<SignedTransaction> {
     throw new MethodNotImplementedError();
```

Comment:
`return Bluebird.reject(new MethodNotImplementedError()).asCallback(callback);`

We should not throw sync errors in async methods, instead we should reject asynchronously.